db/migrate/20190401071349_add_cinema_genre_to_event.rb

```diff
@@ -0,0 +1,5 @@
+class AddCinemaGenreToEvent < ActiveRecord::Migration[5.2]
```

Comment:
Just add genre or style to event, this way we can use it for concerts/theater as well